api/serializers.py

```diff
@@ -77,3 +93,9 @@ class Meta:
             'grading_system',
             'view_grade',
         )
+
+    def create(self, validated_data):
+        learning_module = validated_data.pop('learning_module')
+        new_course = Course(**validated_data)
```

Comment:
Should this be:

new_course = Course(learning_module)

Otherwise, the first element is always missed out.